README.md

```diff
@@ -122,7 +122,7 @@ files normally.
 
 We've kept attribution comments terse, so we ask that you do not actively work
 to remove them from files, especially code. They're a great way for folks to
-learn about Font Awesome.
+learn more about Font Awesome.
```

Comment:
As a non native speaker, I think that this changes the meaning of the sentence

Reply:
but the line i added fits more accurately in at that place.